server/channels/app/channel_bookmark_test.go

```diff
@@ -102,12 +102,16 @@ func TestUpdateBookmark(t *testing.T) {
 			HasPreviewImage: true,
 		}
 
-		th.App.Srv().Store().FileInfo().Save(th.Context, file)
-		defer th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file.Id)
+		_, err := th.App.Srv().Store().FileInfo().Save(th.Context, file)
+		assert.NoError(t, err)
+		defer func() {
+			appErr := th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file.Id)
+			assert.Nil(t, appErr)
```

Comment:
This is an an `error`
```suggestion
			err = th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file.Id)
			assert.NoError(t, err)
```